forge/test/OnChainLiquidator.t.sol

```diff
@@ -0,0 +1,509 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../../contracts/Comet.sol";
+import "../../contracts/CometConfiguration.sol";
+import "../../contracts/liquidator/OnChainLiquidator.sol";
+import "../../contracts/test/SimplePriceFeed.sol";
+
+contract OnChainLiquidatorTest is Test {
+    Comet public comet;
+    OnChainLiquidator public liquidator;
+
+    // contracts
+    address public constant UNISWAP_V3_FACTORY = 0x1F98431c8aD98523631AE4a59f267346ea31F984;
+
+    // assets
+    address public constant CB_ETH = 0xBe9895146f7AF43049ca1c1AE358B0541Ea49704;
+    address public constant COMP = 0xc00e94Cb662C3520282E6f5717214004A7f26888;
+    address public constant DAI = 0x6B175474E89094C44Da98b954EedeAC495271d0F;
+    address public constant LINK = 0x514910771AF9Ca656af840dff83E8264EcF986CA;
+    address public constant ST_ETH = 0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84;
+    address public constant UNI = 0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984;
+    address public constant USDC = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address public constant WBTC = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+    address public constant WETH9 = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public constant WST_ETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+
+    // whales
+    address public constant CB_ETH_WHALE = 0xFA11D91e74fdD98F79E01582B9664143E1036931;
+    address public constant COMP_WHALE = 0x2775b1c75658Be0F640272CCb8c72ac986009e38;
+    address public constant LINK_WHALE = 0xfB682b0dE4e0093835EA21cfABb5449cA9ac9e5e;
+    address public constant UNI_WHALE = 0x1a9C8182C09F50C8318d769245beA52c32BE35BC;
+    address public constant WBTC_WHALE = 0x9ff58f4fFB29fA2266Ab25e75e2A8b3503311656;
+    address public constant WETH_WHALE = 0x2F0b23f53734252Bda2277357e97e1517d6B042A;
+    address public constant WST_ETH_WHALE = 0x10CD5fbe1b404B7E19Ef964B63939907bdaf42E2;
+
+    // wallets
+    address public constant LIQUIDATOR_EOA = 0x5a13D329A193ca3B1fE2d7B459097EdDba14C28F;
+
+    function setUp() public {
+        vm.createSelectFork(string.concat("https://mainnet.infura.io/v3/", vm.envString("INFURA_KEY")));
+
+        SimplePriceFeed wethPriceFeed = new SimplePriceFeed(1e8, 8);
```

Comment:
We haven't deployed our price feeds yet, so I had to create some placeholders.

- [ ] add TODO comment to replace these simplePriceFeeds once the `mainnet/WETH` market is launched